### ISSUE TITLE
sMC - returned values (qf and F-test values)

### ISSUE THREAD

**Issue by gtalckmin** [khliland/plsVarSel#1]
Hi Kristian. A suggestion:
`##plsVarSel::sMC function 
function (pls.object, opt.comp, X, alpha_mc = 0.05) 
{
    b <- pls.object$coefficients[, 1, opt.comp]
    X <- unclass(as.matrix(X))
    n <- dim(X)[1]
    yhat <- X %*% b
    Xhat <- tcrossprod(yhat, b)/crossprod(b)[1]
    Xresidual <- X - Xhat
    SSCregression <- colSums(Xhat^2)
    SSResidual <- colSums(Xresidual^2)
    MSCregression <- SSCregression
    MSResidual <- SSResidual/(n - 2)
    smcF <- MSCregression/MSResidual
    smcFcrit <- qf(1 - alpha_mc, 1, n - 2)
    attr(smcF, "quantile") <- smcFcrit
    smcF
}
` 
I reckon it will not be clear to many users that the sMC function returns the F-values (of each variable) and the critical (threshold F-value). 
In other words, most will assume that the value returned is the p-value of the F-test. 
It would be worthwhile to add to the documentation.

**Comment by khliland** [comment]
Hi gtalckmin,
Thank you for your input and sorry for not catching this for two whole years!
I will improve the documentation on this.
